bundler/sbom/bundler.cdx.json

```diff
@@ -0,0 +1,170 @@
+{
+  "$schema": "http://cyclonedx.org/schema/bom-1.6.schema.json",
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "version": 1,
+  "metadata": {
+    "component": {
+      "type": "library",
+      "name": "bundler",
+      "version": "2.7.0.dev",
+      "purl": "pkg:gem/bundler@2.7.0.dev"
+    }
+  },
+  "components": [
+    {
+      "type": "library",
+      "name": "connection_pool",
+      "version": "2.5.0",
+      "licenses": [
+        {
+          "license": {
+            "id": "MIT"
+          }
+        }
+      ],
+      "description": "Generic connection pool for Ruby",
+      "hashes": [
+        {
+          "alg": "SHA-256",
+          "content": "233b92f8d38e038c1349ccea65dd3772727d669d6d2e71f9897c8bf5cd53ebfc"
+        }
+      ],
+      "purl": "pkg:gem/connection_pool@2.5.0"
```

Comment:
We could consider using https://cyclonedx.org/docs/1.6/json/#components_items_pedigree to describe how we modify the original package for vendoring it, for example, by listing the patches we apply to it.